samples/nrf9160/aws_fota/src/main.c

```diff
@@ -37,9 +37,12 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_LTE_AUTO_INIT_AND_CONNECT),
 
 #if !defined(CONFIG_CLOUD_CLIENT_ID)
 /* Define the length of the IMEI AT COMMAND response buffer */
-#define CGSN_RESP_LEN 19
 #define IMEI_LEN 15
-#define CLIENT_ID_LEN (IMEI_LEN + sizeof("nrf-"))
+#if !defined(CONFIG_USE_NRF_CLOUD)
+#define CLIENT_ID_LEN (sizeof(CONFIG_CLOUD_CLIENT_ID_PREFIX) - 1 + IMEI_LEN)
+#else
+#define CLIENT_ID_LEN (sizeof(CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX) - 1 + IMEI_LEN)
```

Comment:
```suggestion
#define CLIENT_ID_LEN (sizeof(CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX) + IMEI_LEN) 
```
seems to fix what I'm seeing.